Rakefile

```diff
@@ -36,7 +41,7 @@ end
 desc 'Run RuboCop over itself'
 RuboCop::RakeTask.new(:internal_investigation)
 
-task default: %i[documentation_syntax_check spec internal_investigation]
+task default: %i[documentation_syntax_check spec prism_spec internal_investigation]
```

Comment:
Isn’t prism_spec effectively just a duplicate of rspec task when executed from non-prism CI jobs, @koic ? Same in rubocop-rails.
The suggestion is to run the default rake task in the prism ci job, and remove prism_spec. Wdyt?

Reply:
Normally, efforts are made to ensure that `prism_spec` passes locally. If it becomes too troublesome, it might be removed, but currently, having `prism_spec` operational is more convenient. So, I'd like to quickly identify issues related to Prism. Cases where tests cannot pass with Prism can be handled with `unsupported_on: :prism` or `broken_on: :prism`.